src/style/mixins.scss

```diff
@@ -171,3 +171,28 @@
     }
   }
 }
+
+/// Shadow shown over scrollable areas
+@mixin iui-scroll-shadow($background-color: t(iui-color-background-1)) {
+  @include themed {
+    background:
+      // Shadow to maintain background color on whole component
+      linear-gradient(
+        transparent $iui-s,
+        $background-color $iui-s calc(100% - #{$iui-s}),
+        transparent calc(100% - #{$iui-s})
+      ),
+      // Cover top shadow
+      linear-gradient($background-color 0% 100%) center top,
+      // Cover bottom shadow
+      linear-gradient($background-color 0% 100%) center bottom,
+      // Top shadow
+      linear-gradient(180deg, rgba(0, 0, 0, t(iui-opacity-6)) 20%, $background-color) center top,
+      // Bottom shadow
+      linear-gradient(0deg, rgba(0, 0, 0, t(iui-opacity-6)) 20%, $background-color) center bottom;
```

Comment:
The top/bottom shadow would look much better with radial-gradient like in https://codepen.io/chriscoyier/pen/YzXBYvL?editors=1100 but I couldn't figure out how to get the first `linear-gradient`  (used to maintain the background color on the whole component) to work correctly with this. Since it covers everything except the bottom/top 8px (where the scroll shadow is shown), with the radial gradient it doesn't go in the corners so the background color gets messed up in the bottom/top corners. 

![image](https://user-images.githubusercontent.com/88331924/142266440-8a728670-f7b4-4319-8414-94327c949888.png)

Reply:
What have you tried?

I was able to change the body background to `grey` in that codepen and had to make the same change in the linear gradient
![image](https://user-images.githubusercontent.com/9084735/142267857-3e799f38-5ac6-4fec-bde3-3092b34dd52b.png)
![image](https://user-images.githubusercontent.com/9084735/142267982-6fbb7779-7e9e-4d44-a7ab-149e8b1d128c.png)

Reply:
ok I see the problem, it's because the background color and the gradients are applied on the same element. This can be solved by a wrapper div or maybe even a pseudo element

Reply:
I started exploring our alternative options.  I'm thinking we might want to take [a JS based approach](https://javascript.plainenglish.io/how-to-create-horizontal-vertical-scroll-shadows-26aa7a7f00a0#4203).  The reason I'm thinking the CSS only approach won't work is because the scroll shadow appears underneath UI elements.  
_In the screenshots below I turned the shadow opacity up very high so you can better see the issues._
![Screen Shot 2021-12-07 at 12 03 07 PM](https://user-images.githubusercontent.com/849817/145073526-286979f2-410a-4b19-a243-9d2bce8511bf.png)

What I'm thinking might be better is to use `::before` & `::after` to create a block at the top and the bottom of the scrollable area.  I have started exploring this option, but I can't get the blocks to stay at the top and bottom, they want to scroll.
![Screen Recording 2021-12-07 at 12 09 23 PM](https://user-images.githubusercontent.com/849817/145074832-86b05bdf-b54e-4e96-9327-f5b4290a9579.gif)
Here's the SCSS I have right now:
```SCSS
@mixin iui-scroll-shadow($background-color: t(iui-color-background-1)) {
  position: relative;

  &::before,
  &::after {
    content: '';
    position: absolute;
    left: 0;
    right: 0;
    width: 100%;
    height: $iui-baseline;
    pointer-events: none;
    background-repeat: no-repeat;
    background-size: 100% $iui-s;
  }

  &::before {
    top: 0;
    bottom: auto;
    @include themed {
      background: radial-gradient(farthest-side at 50% 0, rgba(0, 0, 0, t(iui-opacity-5)), rgba(0, 0, 0, 0)) center top;
    }
  }

  &::after {
    top: auto;
    bottom: 0;
    @include themed {
      background: radial-gradient(farthest-side at 50% 100%, rgba(0, 0, 0, t(iui-opacity-5)), rgba(0, 0, 0, 0)) center bottom;
    }
  }
}
```

I'm thinking the JS method might be best if we turn on / off the `::before` & `::after` depending on where the scroll bar is.  Here's [a demo I've been looking at](https://docs.telerik.com/kendo-ui/knowledge-base/grid-add-scroll-shadow) that seems to be working.

Reply:
I think with the current wrapper div approach, the scroll shadow still appears underneath UI elements like inputs. Using a pseudo-element would allow placing it on top of all content

Reply:
I just pushed my attempt at adding the scroll shadows with a wrapper div using `::before` & `::after` like in the link Jon posted, but it is not a one size fits all solution. 
It works in information panel and menu:
![scrollOnInfoPanel](https://user-images.githubusercontent.com/88331924/148793128-2ca144f7-47c5-4fae-85d7-29f522ad6c33.gif)
![scrollOnMenu](https://user-images.githubusercontent.com/88331924/148792923-b8879882-97a5-4f5a-a7ff-ec4172d1222e.gif)
But I was not able to get it to work on Table or textarea.
In table the shadows are added and removed correctly but they do not stay at the top/bottom. 
![scrollOnTable](https://user-images.githubusercontent.com/88331924/148791912-a4863da8-d64e-48ef-85d8-58ba71cf0b91.gif)
In textarea the layout gets messed up and the bottom shadow doesn't work.
![scrollOnTextarea](https://user-images.githubusercontent.com/88331924/148794063-acb94609-b63e-45bd-ab2b-ce5ebfb40c22.gif)